pipelines/facebook_ads/fb.py

```diff
@@ -24,80 +24,107 @@
-def notify_on_token_expiration(
-    access_token_expires_at: int = None
-) -> None:
+def notify_on_token_expiration(access_token_expires_at: int = None) -> None:
     """Notifies (currently via logger) if access token expires in less than 7 days. Needs `access_token_expires_at` to be configured."""
     if not access_token_expires_at:
-        logger.warning("Token expiration time notification disabled. Configure token expiration timestamp in access_token_expires_at config value")
+        logger.warning(
+            "Token expiration time notification disabled. Configure token expiration timestamp in access_token_expires_at config value"
+        )
     else:
         expires_at = pendulum.from_timestamp(access_token_expires_at)
         if expires_at < pendulum.now().add(days=7):
-            logger.error(f"Access Token expires in {humanize.precisedelta(pendulum.now() - expires_at)}. Replace the token now!")
+            logger.error(
+                f"Access Token expires in {humanize.precisedelta(pendulum.now() - expires_at)}. Replace the token now!"
+            )
 
 
 @with_config(sections=("sources", "facebook_ads"))
 def debug_access_token(
     access_token: str = dlt.secrets.value,
     client_id: str = dlt.secrets.value,
-    client_secret: str = dlt.secrets.value
+    client_secret: str = dlt.secrets.value,
 ) -> str:
     """Debugs the `access_token` providing info on expiration time, scopes etc. If arguments are not provides, `dlt` will inject them from configuration"""
-    debug_url = f'https://graph.facebook.com/debug_token?input_token={access_token}&access_token={client_id}|{client_secret}'
+    debug_url = f"https://graph.facebook.com/debug_token?input_token={access_token}&access_token={client_id}|{client_secret}"
     response = requests.get(debug_url)
     data: Dict[str, str] = response.json()
 
-    if 'error' in data:
+    if "error" in data:
         raise Exception(f"Error debugging token: {data['error']}")
 
-    return data['data']
+    return data["data"]
 
 
 @with_config(sections=("sources", "facebook_ads"))
 def get_long_lived_token(
     access_token: str = dlt.secrets.value,
     client_id: str = dlt.secrets.value,
-    client_secret: str = dlt.secrets.value
+    client_secret: str = dlt.secrets.value,
 ) -> str:
     """Gets the long lived access token (60 days) from `access_token`. If arguments are not provides, `dlt` will inject them from configuration"""
     exchange_url = f"https://graph.facebook.com/v13.0/oauth/access_token?grant_type=fb_exchange_token&client_id={client_id}&client_secret={client_secret}&fb_exchange_token={access_token}"
     response = requests.get(exchange_url)
     data: Dict[str, str] = response.json()
 
-    if 'error' in data:
+    if "error" in data:
         raise Exception(f"Error refreshing token: {data['error']}")
 
     return data["access_token"]
 
 
-def get_ads_account(account_id: str, access_token: str, request_timeout: float) -> AdAccount:
+def get_ads_account(
```

Comment:
looks like this should go to `helpers.py`. things in `fb.py` are useful utils for the user but actually not used by pipeline code. (we can rename that file to utils.py - and mention this in README)